requirements.txt

```diff
@@ -0,0 +1,4 @@
+prometheus-client
+confuse
+asyncio
+juju == 2.9.11
```

Comment:
why pin to this specific version?

Reply:
We are pinging the version because technically libjuju 3.0 is intended for Juju 3.0. And since we don't know when they are going to drop support for Juju 2.9 entirely, we figured that the safest way to go about this is the ping the most recent version of libjuju 2.9.